src/main/resources/db/changelog/db.changelog-master.yaml

```
@@ -0,0 +1,47 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
+      author: serg.kopervas
+      changes:
+        - createTable:
+            tableName: books
+            columns:
+              - column:
+                  name: id
+                  type: BIGINT
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: title
+                  type: VARCHAR(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: author
+                  type: VARCHAR(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: isbn
+                  type: VARCHAR(255)
+                  constraints:
+                    nullable: false
+                    unique: true
+              - column:
+                  name: price
+                  type: DECIMAL(10, 2)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: description
+                  type: VARCHAR(1024)
+              - column:
+                  name: cover_image
+                  type: VARCHAR(255)
+
+
+  - include:
+      file: db/changelog/changes/add-is-deleted-to-books.yaml
```

replace to `master` changelog file 
```suggestion
  - include:
      file: db/changelog/changes/add-is-deleted-to-books.yaml
```